site/front/poc/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kaart met Weerdata</title>
```

Comment:
het is poc, maar voor later graag andere titel
```suggestion
    <title>Re-Energize 

- [ ] Championship</title>

```